UI/volume-control.cpp

```diff
@@ -423,30 +423,31 @@ void VolumeMeter::setPeakMeterType(enum obs_peak_meter_type peakMeterType)
 	obs_volmeter_set_peak_meter_type(obs_volmeter, peakMeterType);
 	switch (peakMeterType) {
 	case TRUE_PEAK_METER:
-		// For true-peak meters EBU has defined the Permitted Maximum,
-		// taking into account the accuracy of the meter and further
-		// processing required by lossy audio compression.
+		// Here, "true-peak" level according to ITU-R BS.1770, Annex 2.
 		//
-		// The alignment level was not specified, but I've adjusted
-		// it compared to a sample-peak meter. Incidently Youtube
-		// uses this new Alignment Level as the maximum integrated
-		// loudness of a video.
+		// Permitted Maximum Level (PML) = -1.0 dBFS (a)
+		// Alignment Level (AL) = -18 dBFS (b)
 		//
-		//  * Permitted Maximum Level (PML) = -2.0 dBTP
-		//  * Alignment Level (AL) = -13 dBTP
-		setErrorLevel(-2.0);
-		setWarningLevel(-13.0);
+		// a) recommended -1 dB Full Scale (EBU R128)
+		//    and -3dB for possible operator error (EBU 68-2000);
```

Comment:
The -3dB for possible operator error was in reference the quasi-peak meter. The operator error does not count here, nor does it count for the sample peak meter, due to peak-hold meter, this allows the operator to read the peak exactly, the quasi-peak meter does not have this sample peak meter.

EBU R128 Did mention specifically to add -1 dB to the permittted maximum level if the audio would be compressed using mpeg audio encoding (I am guessing for any kind of lossy audio encoding).

I would keep the setErrorLevel to -2.